### PR TITLE
chore: add basic meta tags to blog

### LINKED ISSUE #31: feature: add meta tags to blog
To enable OpenGraph descriptions, pictures etc. we should also add the most important meta tags to the blog landing and post pages. What do you think?

### PR DESCRIPTION
**Pull request by niklasmtj**
This addes the basic meta tags with already available content. Beyond that we will still need to figure out how we would like to proceed with images etc.

fixes #31

### CODE REVIEW

File: routes/blog/[slug].tsx [hunk 1]
```diff
@@ -33,6 +34,15 @@ export default function PostPage(props: PageProps<Post>) {
...
         <title>{post.title}</title>
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content={SITE_NAME} />
+        <meta property="og:locale" content="en" />
+        <meta property="og:title" content={post.title} />
+        <meta property="og:description" content={post.summary} />
+        <meta property="og:url" content={props.url.origin} />
+        <meta name="twitter:title" content={post.title} />
+        <meta name="twitter:description" content={post.summary} />
+        <meta name="description" content={post.summary} />
```
Comment (iuioiua) [inline routes/blog/[slug].tsx:45]: Are we able to add `<meta itemprop="name"...>` and `<meta itemprop="description"... >`?

File: routes/blog/index.tsx [hunk 2]
```diff
@@ -36,6 +37,12 @@ export default function BlogIndexPage(props: PageProps<Post[]>) {
     <>
       <Head>
         <title>Blog</title>
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content={SITE_NAME} />
+        <meta property="og:locale" content="en" />
+        <meta property="og:title" content="Blog" />
+        <meta property="og:url" content={props.url.origin} />
+        <meta name="twitter:title" content="Blog" />
```
Comment (iuioiua) [inline routes/blog/index.tsx:45]: Can we match the number of tags as in `/blog/[slug].tsx`? We can have something like "This is the blog for Deno SaaSKit" for the description.

**Review by iuioiua — CHANGES_REQUESTED** [review]
Thank you! LGTM, except for a couple of suggested additions.

File: routes/blog/index.tsx (hunk 2 above)
Comment (niklasmtj) [reply routes/blog/index.tsx]: Sure, added it in the new PR.

File: routes/blog/[slug].tsx (hunk 1 above)
Comment (niklasmtj) [reply routes/blog/[slug].tsx:45]: Of course, I added it to `/blog/[slug].tsx` and `/blog/index.tsx` respectively

**Review by iuioiua — APPROVED** [review]
LGTM! Thank you, @niklasmtj.

**Pull request merged**